src/sagas/api.js

```diff
@@ -4,79 +4,132 @@ import { call, select } from 'redux-saga/effects';
 import { getToken } from '../selectors/account';
 import { getApiRoot, getTokenType } from '../selectors/env';
 
-export function* getHeaders (authenticationRequired) {
+import * as _ from 'lodash';
+
+const parseOptions = (options) => {
+  let data = null;
+  let customHeaders = undefined;
+  let authenticationRequired = true;
+
+  let hasData = _.has(options, 'data');
```

Comment:
It looks like these should be `const` instead as they don't change.  Alternatively, 
`
if (_.has(options, 'data')) {
  data = _.get(options, 'data');
}

if (_.has(options, 'headers')) {
  customHeaders = _.get(options, 'headers');
}

if (_.has(options, 'authenticationRequired')) {
  authenticationRequired = _.get(options, 'authenticationRequired');
}
`